src/Bicep.Core/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Xml;
+
+public static class DateTimeExtensions
+{
+    public static string ToRFC3339(this DateTime date)
+    {
+        XmlConvert.ToString(date, XmlDateTimeSerializationMode.Utc);
+        return date.ToUniversalTime().ToString("yyyy-MM-dd'T'HH:mm:ss.fffK");
```

Comment:
It seems the custom format specifier can be replaced by the [round-trip formatter specifier](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier).
```suggestion
        return date.ToUniversalTime().ToString("o");
```

Reply:
Thanks.  This does give more digits of precision in fractional seconds but that seems to be allowable.

Reply:
Thanks!